fastly/resource_fastly_service_wasm_v1.go

```diff
@@ -0,0 +1,22 @@
+package fastly
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+// Ordering is important - stored is processing order
+// Some objects may need to be updated first, as they can be referenced by other
+// configuration objects (Backends, Request Headers, etc).
+var wasmService = &BaseServiceDefinition{
+	Type: "wasm",
+	Attributes: []ServiceAttributeDefinition{
+		NewServiceDomain(),
+		NewServiceHealthCheck(),
+		NewServiceBackend(),
+		NewServicePackage(),
+	},
+}
+
+func resourceServiceWasmV1() *schema.Resource {
```

Comment:
Should we rename the use of `ServiceWasm` to `ServiceCompute`?

Reply:
It's an difficult one... The service type in the API is definitely "wasm" so this leaves the answer somewhat ambiguous.